tools/ejections/cmd/main.go

```diff
@@ -77,7 +77,7 @@ func RunScan(ctx *cli.Context) error {
 	if chainState == nil {
 		return errors.New("failed to create chain state")
 	}
-	subgraphApi := subgraph.NewApi(config.SubgraphEndpoint, config.SubgraphEndpoint)
+	subgraphApi := subgraph.NewApi(config.SubgraphEndpoint, config.SubgraphEndpoint, config.SubgraphEndpoint)
```

Comment:
This kind of shows that the NewApi config/constructor is generic for no reasons. Are there ever cases where the different subgraphs don't use the same endpoint? This looks weird that we're just passing the same endpoint 3 times. If we don't ever need the different subgraphs to have different endpoints, then prob best to consolidate them into a single endpoint field like in this config

Reply:
This was weird when I saw it but seems like this specific ejection tool thing only needs the middle one but the this object needs all three. Open to suggestions on refactoring but feel like it's gonna be a big change.

Reply:
oh I see. Fine with just adding a comment then.

Reply:
yea the ejection tool is my work. Passing duplicate urls is a hack since `NewApi` is an abstraction over multiple subgraph clients (ejection only needs one) and I didn't want to justify yak shaving a refactor of 2yo code assumptions built for dependency injection because some esoteric ejection reporting tool needed to call a subgraph 💀